source/nodejs/adaptivecards-designer/src/app.ts

```diff
@@ -28,10 +27,19 @@ function monacoEditorLoaded() {
         function (e) {
             scheduleCardRefresh();
         });
-    document.querySelector(".monaco-editor").insertAdjacentHTML("afterbegin", "<div class='jsonEditorHost-title'>JSON</div>");
+    const jsonSectlionTitle = `
+    <div class='bullet__wrapper'>
+        <span class='bullet__title'>JSON</span>
+        <span class='bullet js-host-json__bullet'>
+            <span class='bullet__icon js-host-json__icon'></span>
+            <span class='bullet__description js-host-json__description'>Hide</span>
+        </span>
+    </div>
+    `;
+    document.querySelector(".monaco-editor").insertAdjacentHTML("afterbegin", jsonSectlionTitle);
```

Comment:
Maybe my feedback wasn't clear: we shouldn't rely on finding the ".monaco-editor" selector. The header for the JSON editor can and should be declared in the app's HTML file instead of being dynamically inserted in code.

Reply:
In progress.

Reply:
In progress.